File: Code/new_fitervari_backend/src/main/java/at/htl/model/NewsLetter.java
package at.htl.model;


import javax.persistence.*;

@Entity
@Table(name = "news_letter")
public class NewsLetter {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String title;

    private String body;

    private String imageUrl;

    //region GetterSetter
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
    //endregion
}

File: Code/fitervari_backend/src/main/java/at/htl_leonding/model/NewsLetter.kt
package at.htl_leonding.model

import javax.persistence.*

@Entity
@Table(name = "news_letter")
class NewsLetter(

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    @Column(name = "title")
    var title: String,
    @Column(name = "body")
    var body: String,
    @Column(name = "imageUrl")
    var imageUrl: String

) {
    fun copyValues(other: NewsLetter) {
        this.title = other.title
        this.body = other.body
        this.imageUrl = other.imageUrl
    }
}